Remotion/Web/Development.WebTesting/BrowserSession/BrowserLogEntry.cs

```diff
@@ -17,6 +17,7 @@
 using System;
```

Comment:
please restore the NotNull. We did this in web testing to provide maximum support for TA engenineers.